baybe/searchspace/continuous.py

```diff
@@ -277,9 +277,12 @@ def comp_rep_columns(self) -> tuple[str, ...]:
     @property
     def comp_rep_bounds(self) -> pd.DataFrame:
         """The minimum and maximum values of the computational representation."""
+        from baybe.utils.numerical import DTypeFloatNumpy
```

Comment:
Is the lazy import here really necessary, i.e. can't we do a top-level import instead?

Reply:
Original reason was because I was unsure if this needs to be the Numpy or Torch import I think. But will move it to the top 👍

Reply:
Changed.